### PR TITLE
ktlint has been setup

### PR DESCRIPTION
**Pull request by gangamamulya**
# Food sharing application
An android Kotlin application to connect food donors such restaurants and individuals with the one in need
# Proposed Changes

Ktlint setup has been done.

# Checklist

[ ] This has been tested
[ ] Self-documenting code
[ ] Code is linted

# Instructions to review

1. Install Android studio
4. Unzip and run the code in android studio.

### CODE REVIEW

**Review by morganlutz — APPROVED** [review]
🎉

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.{java,kt,kts,scala,rs,xml,kt.spec,kts.spec}]
+indent_size = 4
+
+[*.{kt,kts}]
+ij_kotlin_imports_layout=*
+ij_kotlin_allow_trailing_comma=true
```
Comment (morganlutz) [inline .editorconfig:18]: I saw that `super-linter` is failing on some files due to trailing commas. I'm not sure how to modify the rules that `super-linter` follows, or how to make sure it stays up to date with the latest version of ktlint. I recommend replacing that with something you can integrate and run locally, and that allows custom rules (like ktlint does with .editorconfig).

There are a few different gradle plugins for projects out there that do more than just check Kotlin code if you want something more than what ktlint provides on its own.

**Comment by gangamamulya** [comment]
Hi Morgan,

Since I’m already using  ktlint can i continue using that alone and remove
super linter?




Thanks and regards
Amulya Gangam

**Comment by morganlutz** [comment]
That’s fine with me!

Morgan

**Pull request merged**